proxy/src/main/java/com/wavefront/agent/data/LogDataSubmissionTask.java

```diff
@@ -65,6 +67,14 @@ Response doExecute() {
     return api.proxyLogs(AGENT_PREFIX + proxyId.toString(), logs);
   }
 
+  @Override
+  protected TaskResult handleStatus429() {
+    Metrics.newCounter(
+            new MetricName(entityType + "." + handle, "", "failed" + ".ingestion_limit_reached"))
+        .inc(this.weight());
+    return TaskResult.REMOVED;
```

Comment:
Is ok to drop logs on a 429 ? Why not retry ?

Reply:
A 429 from VRLIC means that the daily ingestion limit has been reached

Reply:
So, that's not coming from wavefront ?

Reply:
Correct, its coming from the VRLIC backend